src/richie/apps/search/utils/viewsets.py

```diff
@@ -0,0 +1,19 @@
+"""
+Helpers to enable reuse for needs that are shared between viewsets.
+"""
+
+
+class ViewSetMeta:
```

Comment:
you can set it on self._meta on the viewset but don't call this calss Meta because Meta classes are something else...
Django is kind of misusing this word but it has become a common practice in other project to attach options to a set._meta attribute. But it is meta as in "metadata", not as in "metaclass".
Maybe you could call this class ViewSetOptions...

Reply:
I just went for `ViewSetMetadata`, that way it can't be confused with a metaclass :)